src/2G84.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Literal modifers
+  from: '@ingydotnet'
+  tags: literal scalar
+  fail: true
+  yaml: |
+    --- |0
+  tree: |
+    +STR
+     +DOC ---
+
+- fail: true
+  yaml: |
+    --- |10
+  tree: |
+    +STR
+     +DOC ---
+      =VAL |
+     -DOC
+    -STR
```

Comment:
these 3 lines should be removed

Reply:
Actually the whole `tree` entry can be removed then.